analysis/io.py

```diff
@@ -0,0 +1,68 @@
+import bionlp.utils.monitoring as mo
+import pickle
+import os
+import numpy as np
+
+
+SOURCE_DIR = "data/logs"
+
+
+def load_monitoring(filename="monitor_2017-06-"):
+    accuracy = {mo.TYPE_VALIDATION: [], mo.TYPE_TRAINING: []}
+    loss = {mo.TYPE_VALIDATION: [], mo.TYPE_TRAINING: []}
+    for file in os.listdir(SOURCE_DIR):
+        if file.startswith(filename) and file.endswith('.pkl'):
+            pkl = pickle.load(open(os.path.join(SOURCE_DIR, file), 'rb'))
+            accuracy[mo.TYPE_VALIDATION].append(pkl[mo.TYPE_VALIDATION][mo.METRIC_ACC])
+            accuracy[mo.TYPE_TRAINING].append(pkl[mo.TYPE_TRAINING][mo.METRIC_ACC])
+
+            loss[mo.TYPE_VALIDATION].append(pkl[mo.TYPE_VALIDATION][mo.METRIC_LOSS_TOT])
+            loss[mo.TYPE_TRAINING].append(pkl[mo.TYPE_TRAINING][mo.METRIC_LOSS_TOT])
+
+    return {mo.METRIC_ACC: accuracy, mo.METRIC_LOSS_TOT: loss}
+
+
+def parse_eval_line(line=""):
```

Comment:
our eval files are now .pkl files with confusion matrices instead of .txt files. The precision, recall, and f1 score need to be calculated from the confusion matrix.